spec/forms/refunds/applicant_form_spec.rb

```diff
@@ -204,10 +210,10 @@ module Refunds
         expect(applicant_form.applicant_date_of_birth).to be value
       end
 
-      it 'stores the provided ActionController::Parameters if the date is invalid' do
-        value = ActionController::Parameters.new('day' => '32', 'month' => '15', "year" => '1985')
+      it 'stores the provided ActionController::Parameters as a hash if the date is invalid' do
+        value = ActionController::Parameters.new('day' => '32', 'month' => '15', "year" => '1985').freeze
```

Comment:
Why are you freezing this?

Reply:
To be sure nothing changes it - I was concerned that the underlying code might modify the existing params rather than convert to a new set or a hash